lib/plugins/filter/after_render/external_link.js

```diff
@@ -51,10 +43,13 @@ function externalLinkFilter(data) {
   if (config.external_link === false || config.external_link.enable === false
     || config.external_link.field !== 'site') return;
 
+  config.external_link.exclude = Array.isArray(config.external_link.exclude) ? config.external_link.exclude
+    : [config.external_link.exclude];
+
   data = data.replace(/<a.*?(href=['"](.*?)['"]).*?>/gi, (str, hrefStr, href) => {
-    if (/target=/gi.test(str) || !isExternal(href, config)) return str;
+    if (str.includes('target=') || !isExternal(href, config)) return str;
```

Comment:
`String.includes()` is case-sensitive, so this is not equivalent to the `i` regexp flag, need `str.toLowerCase().includes()`. Hopefully `String.toLowerCase()` is not expensive.

Reply:
According the [benchmark](https://runkit.com/sukkaw/5dc6398fcb31010019d500e6), `test()` is faster. I will revert `includes()` then,